x-pack/legacy/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -5,18 +5,38 @@
  */
 
 import { I18nProvider } from '@kbn/i18n/react';
-import React from 'react';
-
+import React, { useState } from 'react';
+import { EuiGlobalToastList, Toast } from '@elastic/eui';
+import uuid from 'uuid';
 import { EditorFrameInstance } from '../types';
 import { NativeRenderer } from '../native_renderer';
 
 export function App({ editorFrame }: { editorFrame: EditorFrameInstance }) {
+  const [state, setState] = useState<Toast[]>([]);
+
   return (
     <I18nProvider>
       <div>
-        <h1>Lens</h1>
-
-        <NativeRenderer render={editorFrame.mount} nativeProps={undefined} />
+        <EuiGlobalToastList
+          toasts={state}
+          dismissToast={toast => setState(state.filter(t => t !== toast))}
+          toastLifeTimeMs={6000}
+        />
```

Comment:
These toasts are coupled specifically to loading errors, so if there is some kind of issue across different parts of the app, wouldn't we want all the errors to be combined? I think this is why we have a shared toastService.

Reply:
Hm. I was wondering if we had something like that. I was kinda surprised that we didn't, but I couldn't find it. Looks like it's in 'ui/notify'?